packages/navigator/src/browser/navigator-tree.ts

```diff
@@ -25,7 +25,6 @@ import { FileNavigatorFilter } from './navigator-filter';
 export class FileNavigatorTree extends FileTree {
 
     @inject(FileNavigatorFilter) protected readonly filter: FileNavigatorFilter;
-
```

Comment:
@Anasshahidd21 can you revert changes made to the following file? (they are unnecessary)